bundles/binding/org.openhab.binding.http/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="http"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>HTTP Binding</name>
+    <description>Send updates and commands to and poll updates from HTTP servers.</description>
```

Comment:
I think it's sensible to leave it out when authorship isn't clear, but maybe also mention known authors just in case.  @teichsta, would you be the rightful author or anyone else?